tree/ntuple/v7/src/RNTupleDescriptor.cxx

```diff
@@ -663,32 +663,43 @@ ROOT::Experimental::RNTupleDescriptor::CreateModel(const RCreateModelOptions &op
    return model;
 }
 
-ROOT::Experimental::RNTupleDescriptor ROOT::Experimental::RNTupleDescriptor::Clone() const
+ROOT::Experimental::RNTupleDescriptor ROOT::Experimental::RNTupleDescriptor::CloneSchema() const
```

Comment:
`CloneSchema` should be exercised in a unit test, in particular which fields are not expected to be there